ENIGMAsystem/SHELL/Graphics_Systems/General/GSvertex.cpp

```diff
@@ -41,7 +41,7 @@ std::unordered_map<size_t, int> vertexFormatCache;
 
 // current vertex format being specified
 // NOTE: this is not reset until the next vertex_format_begin
-// NOTE: this uses the stack until vertex_format_end to speed up creation
+// NOTE: this is allocated once and not a pointer to avoid reallocation
```

Comment:
A pointer wouldn't cause reallocation; calling new causes reallocation. 😛

Reply:
How about saying that it uses automatic storage duration to avoid multiple allocations?

Reply:
Alright: eee2b925ad233de1ef5fe902b9b0ca9092192363

You said that's better on Discord, so there we go.